mvc/views/tag-helpers/authoring.rst

```diff
@@ -1,8 +1,376 @@
-.. include:: /../common/stub-topic.txt
-
-|stub-icon| Authoring Tag Helpers
+Authoring Tag Helpers
 ===========================================
 
-.. include:: /../common/stub-notice.txt
+By `Rick Anderson`_
+
+    - `Getting started with Tag Helpers`_
```

Comment:
Changed, although the editor disagrees (two of them, both said there is no wrong way and UC is fine).